### ISSUE TITLE
Could not establish to remote after updating win10 to 10.0.18362

### ISSUE THREAD

**Issue by pcshih** [microsoft/vscode-remote-release#1152]
- VSCode Version: 1.38.0-insider (user setup)
- Local OS Version:  Windows_NT x64 10.0.18362
- Remote OS Version: ubuntu 18.04.01 LTS
- Remote Extension/Connection Type: SSH

Steps to Reproduce:

1.  clicking the Connect to Host in New Window button(before update to win10 10.0.18362, it will open new window and connect well)
2. get the issues below
it will pop up a cmd(did not pop before updating win10)
![cmd](https://user-images.githubusercontent.com/48461528/62922399-dfb34280-bddd-11e9-88fa-3ca1d00a3171.PNG)
![vscode output](https://user-images.githubusercontent.com/48461528/62922415-e641ba00-bddd-11e9-890b-90f689f71e68.PNG)
![error window](https://user-images.githubusercontent.com/48461528/62928286-d9779300-bdea-11e9-9072-4f232d96e93a.PNG)

Thank you very much.

**Comment by eric-mu** [comment]
I got the same issue as you :(

**Comment by eric-mu** [comment]
Hey, I got the same issue, but I fixed it. You need to make sure there is no '(' or ')' in your host name in configuration file. Try to use only letters and numbers in your host name.

**Comment by pcshih** [comment]
There is not any '(' or ')' in my host name in configuration file.

Host "ip"
    HostName "ip"
    User PTe
    IdentityFile ~/.ssh/id_rsa

**Comment by eric-mu** [comment]
Maybe you could try to delete the quotation marks？
My configuration is:

Host AI-inter02-207
    HostName 172.16.0.207
    User ppz
    IdentityFile C:\Softwares\VSCode\ssh-key\id_rsa

But it will not work if I modify to:

Host AI-inter02(207)
    HostName 172.16.0.207
    User ppz
    IdentityFile C:\Softwares\VSCode\ssh-key\id_rsa

**Comment by pcshih** [comment]
@eric-mu seems not work...
Host test
    HostName 192.168.0.1
    User PTe
    IdentityFile C:\Users\ec131b\.ssh\id_rsa

**Comment by eric-mu** [comment]
I am sorry to hear that...
Maybe it is different between our issues.
Hope that you can fix it as soon as possible.

**Comment by pcshih** [comment]
@eric-mu 
I hope so......

**Comment by rafipiccolo** [comment]
in vscode settings :
search conpty
and uncheck it

Now it works. although i don't know what it means.

**Comment by roblourens** [comment]
@pcshih if you connect with ssh in a normal windows command prompt, does it open a new window?

**Comment by pcshih** [comment]
@roblourens 
No, it does not open a new window...

@rafipiccolo 
Thank you very much.

**Comment by roblourens** [comment]
So disabling conpty fixed it? FYI @Tyriar I might have pinged you for that one already in another issue.

**Comment by Tyriar** [comment]
You should uncheck this and turn conpty back on:

![image](https://user-images.githubusercontent.com/2193314/63452899-464ae880-c3fc-11e9-957e-ed0f3b14dc15.png)

/duplicate https://github.com/microsoft/vscode/issues/73790

**Comment by pcshih** [comment]
Thank you very much @Tyriar

**Comment by Rechart41** [comment]
Hello,
I also have this problem despite your tips..
![Annotation 2019-08-26 160125](https://user-images.githubusercontent.com/28261182/63696400-e9e03280-c81a-11e9-96ef-8211002b71f7.png)

**Comment by Rechart41** [comment]
@roblourens Oh.. I'm sorry ... I opened a new post https://github.com/microsoft/vscode-remote-release/issues/1249
I can connect to my server with Putty ...

**Comment by rafipiccolo** [comment]
Could you please explain briefly the difference between conpty / winpty ?
why is one better than the other ?

It feels strange to disable something windows 10 activated by default on every machine.

will it be added to the vscode remote tutorial ?

**Comment by Tyriar** [comment]
@rafipiccolo don't disable conpty, uncheck the box as outlined in https://github.com/microsoft/vscode-remote-release/issues/1152#issuecomment-523558088

TLDR, winpty is the legacy third-party system, conpty is the new first-party one.
